src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -341,6 +333,44 @@ private EtwEventSource()
             this.WriteEvent(220, TaskHub, AppName, SlotName, FunctionName, InstanceId, RequestingInstance, RequestId, FunctionType, ExtensionVersion, IsReplay);
         }
 
+        [Event(221, Level = EventLevel.Informational)]
+        public void OperationCompleted(
+            string TaskHub,
+            string AppName,
+            string SlotName,
+            string FunctionName,
+            string InstanceId,
+            string OperationId,
+            string OperationName,
+            string Input,
+            string Output,
+            double Duration,
+            string FunctionType,
+            string ExtensionVersion,
+            bool IsReplay)
+                {
+            this.WriteEvent(206, TaskHub, AppName, SlotName, FunctionName, InstanceId, OperationId, OperationName, Input ?? "(null)", Output ?? "(null)", Duration, FunctionType, ExtensionVersion, IsReplay);
```

Comment:
Change `206` to `221` here.

And for the sake of tidiness, we should probably fix the indentation of the `{` right above this line. :)

Reply:
yes.